config/config.yaml

```
@@ -7,3 +7,28 @@ framework:
     messenger:
         buses:
             sylius_product_bundle.command_bus: ~
+
+sylius_grid:
+    grids:
+        sylius_admin_product:
+            actions:
+                main:
+                    create:
+                        type: links
+                        label: sylius.ui.create
+                        options:
+                            class: primary
+                            icon: "tabler:plus"
+                            header:
+                                icon: cube
+                                label: sylius.ui.type
+                            links:
```

I wonder if we need this part or if it will be merged with the config from Sylius? 🤔 
```suggestion
                    create:
                        options:
                            links:
```